.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,27 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+    # will start the job at 3 every monday (UTC)
+    - cron: "0 3 * * 1"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Target branch to create requirements PR against"
+        required: true
+        default: 'master'
```

Comment:
```suggestion
        default: $default-branch
```

Reply:
$default-branch is for workflow-templates only